curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-363-bzier-curves.md

```diff
@@ -8,37 +8,47 @@ dashedName: problem-363-bzier-curves
 
 # --description--
 
-A cubic Bézier curve is defined by four points: P0, P1, P2 and P3.
+A cubic Bézier curve is defined by four points: $P_0$, $P_1$, $P_2$ and $P_3$.
 
-The curve is constructed as follows: On the segments P0P1, P1P2 and P2P3 the points Q0,Q1 and Q2 are drawn such that P0Q0 / P0P1 = P1Q1 / P1P2 = P2Q2 / P2P3 = t (t in \[0,1]). On the segments Q0Q1 and Q1Q2 the points R0 and R1 are drawn such that Q0R0 / Q0Q1 = Q1R1 / Q1Q2 = t for the same value of t. On the segment R0R1 the point B is drawn such that R0B / R0R1 = t for the same value of t. The Bézier curve defined by the points P0, P1, P2, P3 is the locus of B as Q0 takes all possible positions on the segment P0P1. (Please note that for all points the value of t is the same.)
+The curve is constructed as follows:
 
-At this (external) web address you will find an applet that allows you to drag the points P0, P1, P2 and P3 to see what the Bézier curve (green curve) defined by those points looks like. You can also drag the point Q0 along the segment P0P1.
+<img class="img-responsive center-block" alt="construction of Bézier curve" src="https://cdn.freecodecamp.org/curriculum/project-euler/bzier-curves.png" style="background-color: white; padding: 10px;">
 
-From the construction it is clear that the Bézier curve will be tangent to the segments P0P1 in P0 and P2P3 in P3.
+On the segments $P_0P_1$, $P_1P_2$ and $P_2P_3$ the points $Q_0$,$Q_1$ and $Q_2$ are drawn such that $\frac{P_0Q_0}{P_0P_1} = \frac{P_1Q_1}{P_1P_2} = \frac{P_2Q_2}{P_2P_3} = t$, with $t$ in [0,1].
 
-A cubic Bézier curve with P0=(1,0), P1=(1,v), P2=(v,1) and P3=(0,1) is used to approximate a quarter circle. The value v > 0 is chosen such that the area enclosed by the lines OP0, OP3 and the curve is equal to π/4 (the area of the quarter circle).
+On the segments $Q_0Q_1$ and $Q_1Q_2$ the points $R_0$ and $R_1$ are drawn such that $\frac{Q_0R_0}{Q_0Q_1} = \frac{Q_1R_1}{Q_1Q_2} = t$ for the same value of $t$.
 
-By how many percent does the length of the curve differ from the length of the quarter circle? That is, if L is the length of the curve, calculate 100 × L − π/2π/2Give your answer rounded to 10 digits behind the decimal point.
+On the segment $R_0R_1$ the point $B$ is drawn such that $\frac{R_0B}{R_0R_1} = t$ for the same value of $t$.
+
+The Bézier curve defined by the points $P_0$, $P_1$, $P_2$, $P_3$ is the locus of $B$ as $Q_0$ takes all possible positions on the segment $P_0P_1$. (Please note that for all points the value of $t$ is the same.)
+
+At this (external) web address you will find an applet that allows you to drag the points $P_0$, $P_1$, $P_2$ and $P_3$ to see what the Bézier curve (green curve) defined by those points looks like. You can also drag the point $Q_0$ along the segment $P_0P_1$.
```

Comment:
```suggestion
You will find an applet at this (external) web address that allows you to drag the points $P_0$, $P_1$, $P_2$ and $P_3$ to see what the Bézier curve (green curve) defined by those points looks like. You can also drag the point $Q_0$ along the segment $P_0P_1$.
```

Reply:
Welp, this whole part is no longer in the problem description https://projecteuler.net/problem=363. I'm gonna remove it. Sorry!